src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,23 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+interface TodoInfoProps {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<TodoInfoProps> = ({ todo }) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={cn(
+        { 'TodoInfo TodoInfo--completed': todo.completed },
+        { TodoInfo: !todo.completed },
+      )}
+    >
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+      {todo.user && <UserInfo user={todo.user} />}{' '}
```

Comment:
You dont need a space in the end of the line
```suggestion
      {todo.user && <UserInfo user={todo.user} />}
```